react/src/RestrictedFooter/RestrictedCompactFooter.tsx

```diff
@@ -15,18 +15,21 @@ interface RestrictedCompactFooterProps extends RestrictedFooterVariantProps {
 export const RestrictedCompactFooter = ({
   appName,
   footerIconLink,
-  textColorScheme,
   footerLinks,
   compactMonochromeLogos,
   brandLogo,
   appLink,
+  bgColor: backgroundColor = 'primary.100',
```

Comment:
nit: should we use `bg` over anything else? `bg` is more modern and is also used over `bgColor` across the app. also don't see the need to rename this prop